products/redhat.md

```diff
@@ -11,36 +11,74 @@ releasePolicyLink: https://access.redhat.com/support/policy/updates/errata
 activeSupportColumn: true
 releaseDateColumn: true
 releaseColumn: false
-sortReleasesBy: 'release'
+sortReleasesBy: 'cycleShortHand'
 releaseLabel: "RHEL __RELEASE_CYCLE__"
+LTSLabel: "<abbr title='Extended Life Cycle Support'>ELS</abbr>"
 releases:
   - releaseCycle: "9"
     release: 2022-05-18
     support: 2027-05-31
     eol: 2032-05-31
     latest: "9.0"
+    cycleShortHand: 90
+  - releaseCycle: "9" # ELS Entry
+    release: 2032-05-31
+    support: 2034-05-31
+    eol: 2034-05-31
+    lts: true 
+    cycleShortHand: 91
   - releaseCycle: "8"
     release: 2019-05-01
     support: 2024-05-31
     eol: 2029-05-31
     latest: "8.6"
+    cycleShortHand: 80
+  - releaseCycle: "8" # ELS Entry
+    release: 2029-05-31
+    support: 2031-05-31
+    eol: 2031-05-31
+    lts: true 
+    cycleShortHand: 81
   - releaseCycle: "7"
     release: 2014-06-10
     support: 2019-12-31
     eol: 2024-06-30
     latest: "7.9"
+    cycleShortHand: 70
+  - releaseCycle: "7" # ELS Entry
+    release: 2024-06-30
```

Comment:
same

Reply:
These dates are correct, I'm not happy with it either but the fact is ELS is part of the support cycle of its corresponding release. RHEL 7 ELS is still RHEL 7, just a third phase of support which is distinctly different than the other two, which merits its own entry one way or the other.

The ideal solution would be to be able to add an additional column but from my understanding this is not possible as pointed out in the issue this PR attempts to solve: https://github.com/endoflife-date/endoflife.date/issues/699#issuecomment-1023517852

We could make the release dates for ELS the same as original, but I feel like that could result in more confusion.

Reply:
Let's pause a bit on this PR, I'll attempt to see if I can add a `extendedSupport` column (since that seems like the main request for a fourth column).

We can add the text bits for now for clarity, and work on the extra column?

Reply:
> We can add the text bits for now for clarity, and work on the extra column?

Would commenting the releases out be sufficient, remove them completely or do we just open a new PR with just the text bits?

Reply:
>  I'll attempt to see if I can add a `extendedSupport` column (since that seems like the main request for a fourth column).

This would be greatly appreciated. Would make correctly representing Ubuntu ESM not be a similar mess as this, same goes for Debian and possibly Windows (although that would require a lot more fields)

Reply:
Tried it on https://github.com/endoflife-date/endoflife.date/pull/2157.

Reply:
@Evernow, the new `extendedSupport` column is now available. Could you have a look at https://endoflife.date/rhel and https://endoflife.date/ubuntu and let us know if it is OK ?

If you think of other products that could benefit from this new column you can comment #2246.

Reply:
> @Evernow, the new `extendedSupport` column is now available. Could you have a look at https://endoflife.date/rhel and https://endoflife.date/ubuntu and let us know if it is OK ?

RHEL looks fine with this: https://github.com/endoflife-date/endoflife.date/pull/2253

I'll take a look at Ubuntu in a bit, on a side note thank you very much for your hard work, greatly appreciated! This PR can be closed